src/ui/server/controllers/insurance/export-contract/how-much-the-agent-is-charging/index.ts

```diff
@@ -170,5 +172,17 @@ export const post = async (req: Request, res: Response) => {
     }
   }
 
-  return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`);
+  try {
+    const saveResponse = await mapAndSave.exportContractAgentServiceCharge(payload, application);
+
+    if (!saveResponse) {
+      return res.redirect(PROBLEM_WITH_SERVICE);
+    }
+
+    return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`);
+  } catch (error) {
+    console.error('Error updating application - export contract - how much the agent is charging %O', error);
```

Comment:
should this be lowercase? %o?

Reply:
I am not sure what the UKEF preference is.

We have 611 instances in EXIPS of `%O`, 2 instances of it being in lowercase.

If it needs to change we should do this in a different PR as it affects 405 files.

Reply:
Lowercase %o is the preference. I have created EMS-3894 to tackle this.